modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/group/src/customer-portal/fragments/information-banner/index.html

```diff
@@ -0,0 +1,19 @@
+<div class="alert-fluid alert-warning cp-information-banner">
+	<div class="cp-banner-content">
+		<div class="alert-indicator cp-icon-content">
+			[@clay.icon symbol="warning" /]
+		</div>
+
+		<p>
+			${languageUtil.get(locale,
+			"liferays-commercial-offerings-have-been-renamed-from-lxc-to-liferay-saas-lxc-sm-to-liferay-paas-and-dxp-to-liferay-self-hosted",
+			"Liferay's commercial offerings have been renamed from LXC to Liferay SaaS, LXC - SM to Liferay PaaS, and DXP to Liferay Self-Hosted.")}
+		</p>
+	</div>
+
+	[#if configuration.dismissible]
+	<button type="button" class="close" aria-label="Close">
```

Comment:
i think `<button>` should be indented since its inside an if

Reply:
Done.